bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/handler/VehicleHandler.java

```diff
@@ -381,8 +390,29 @@ private void actionHeater(String action, Boolean start) {
         if (bridgeHandler != null) {
             if (activeOptions.containsKey(action)) {
                 try {
-                    String command = start ? "start" : "stop";
-                    bridgeHandler.postURL(SERVICE_URL + "vehicles/" + configuration.vin + "/heater/" + command, null);
+                    if (action.contains(REMOTE_HEATER)) {
+                        String command = start ? "start" : "stop";
+                        if (command.contains("start")) {
+                            String address = SERVICE_URL + "vehicles/" + configuration.vin + "/heater/" + command;
+                            String json = new String("{}");
+                            bridgeHandler.postURL(address, json);
+                        } else if (command.contains("stop")) {
+                            String address = SERVICE_URL + "vehicles/" + configuration.vin + "/heater/" + command;
+                            bridgeHandler.postURL(address, null);
+                        }
+                    } else if (action.contains(PRECLIMATIZATION)) {
+                        String command = start ? "start" : "stop";
+                        if (command.contains("start")) {
```

Comment:
Simple remove the if and else construction and replace:
```
bridgeHandler.postURL(address, start? "{}" : null);

Reply:
Corrected in new commit.